lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/audio_output.rb

```diff
@@ -0,0 +1,70 @@
+class Mic
```

Comment:
This looks like it was copied and pasted from `lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/mic.rb`, does this have any relation to this new module?

Reply:
Whoops, fixing.